website/write_footer.php

```diff
@@ -58,7 +58,7 @@ function write_footer_new($update_date)
 		print("	 </td>\n");
 		print("  <td align=\"right\">\n");
 		print("<a href=\"http://www.dias.ie//\"><img src=\"./common_files/images/logos/dias_logo.jpg\" align=right border=0 height=40></a>\n");
-		print("<a href=\"http://www.tcd.ie/\"><img src=\"./common_files/images/logos/TCD-logo-wide_small.jpg\" align=right border=0 height=40></a>\n");
+		print("<a href=\"http://www.tcd.ie/\"><img src=\"./common_files/images/logos/TCD-logo-wide_small-new.jpg\" align=right border=0 height=40></a>\n");
```

Comment:
overwrite the old file with the new images, instead of keeping both in the repository.